clang/test/Driver/clang-offload-bundler-bc-archive-support-win.cpp

```diff
@@ -10,105 +11,105 @@
-// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-spir64_gen               -output=%t_list4.txt
-// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-spir64_x86_64            -output=%t_list5.txt
-
-// Backtick not supported on Windows
-// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_A.o `cat %t_list1.txt`
-// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_B.o `cat %t_list2.txt`
-// RUN_disabled_because_backtick_not_supported: cmp %t1.bc           `cat %t_list3.txt`
-// RUN_disabled_because_backtick_not_supported: cmp %t2.bc           `cat %t_list4.txt`
-// RUN_disabled_because_backtick_not_supported: cmp %t3.bc           `cat %t_list5.txt`
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Test archive unbundling
+// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=sycl-spir64-unknown-unknown   -output=%t_list1.txt
+// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-x86_64-pc-windows-msvc   -output=%t_list2.txt
+// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-spir64-unknown-unknown   -output=%t_list3.txt
+// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-spir64_gen               -output=%t_list4.txt
+// RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=host-spir64_x86_64            -output=%t_list5.txt
+
+// Backtick not supported on Windows
+// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_A.o `cat %t_list1.txt`
+// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_B.o `cat %t_list2.txt`
+// RUN_disabled_because_backtick_not_supported: cmp %t1.bc           `cat %t_list3.txt`
+// RUN_disabled_because_backtick_not_supported: cmp %t2.bc           `cat %t_list4.txt`
+// RUN_disabled_because_backtick_not_supported: cmp %t3.bc           `cat %t_list5.txt`
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 // TEST4
-// Test archive unbundling for multiple targets
+// Test archive unbundling for multiple targets
 // RUN: clang-offload-bundler -unbundle -type=aoo -input=%t_bundled.a -targets=sycl-spir64-unknown-unknown,host-spir64_gen -output=%t_listA.txt -output=%t_listB.txt
-// Backtick not supported on Windows
-// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_A.o `cat %t_listA.txt`
-// RUN_disabled_because_backtick_not_supported: cmp %t2.bc           `cat %t_listB.txt`
+// Backtick not supported on Windows
+// RUN_disabled_because_backtick_not_supported: cmp %t_unbundled_A.o `cat %t_listA.txt`
+// RUN_disabled_because_backtick_not_supported: cmp %t2.bc           `cat %t_listB.txt`
 
-#include <sycl/sycl.hpp>
+#include <sycl/sycl.hpp>
```

Comment:
All of the non-visual changes are due to a cleanup of some carriage return characters.